README.md

```diff
@@ -18,8 +18,8 @@ views or policies of the Department of Defense or the U.S. Government.
 
 # Solver Compatibility
 
-| Feature                               | ABC | Boolector | CVC4   | Dreal | STP      | Yices    | Z3              |
-|---------------------------------------|-----|-----------|--------|-------|----------|----------|-----------------|
-| Supported                             | yes | 3.2.2, ?  | 1.8, ? | yes   | 2.3.3, ? | 2.6.4, ? | 4.8.8 -- 4.8.14 |
-| goal timeouts                         | ?   | yes       | yes    | ?     | yes      | yes      | ! 4.8.12        |
-| strings with unicode and escape codes | ?   | ?         | yes    | ?     | ?        | ?        | >= 4.8.12       |
+| Feature                               | ABC | Boolector   | CVC4      | Dreal | STP         | Yices    | Z3                   |
+|---------------------------------------|-----|-------------|-----------|-------|-------------|----------|----------------------|
+| Supported                             | yes | >= 3.2.0, ? | >= 1.8, ? | yes   | >= 2.3.3, ? | 2.6.x, ? | 4.8.8 -- 4.8.14      |
+| goal timeouts                         | ?   | yes         | yes       | ?     | yes         | yes      | ! (4.8.11 or 4.8.12) |
+| strings with unicode and escape codes | ?   | ?           | >= 1.8    | ?     | ?           | ?        | >= 4.8.11            |
```

Comment:
I wonder if we should make a special mention of Z3 4.8.10 due to GaloisInc/cryptol#1107?

Reply:
Is there any way to extract the what4 formula from `negshift.icry` to put into a test here?  I think it would be good to update the matrix, but having a supporting test in this same repo would be even better.

Reply:
https://github.com/Z3Prover/z3/issues/5106#issue-832248620 contains an `.smt2` file that tickles the same issue, but I haven't spent the time to turn this into a What4 expression.

Reply:
On second thought, this wouldn't make for a good `what4` test case anyway. The original Cryptol test case that https://github.com/Z3Prover/z3/issues/5106#issue-832248620 was distilled from made essential use of the SBV backend, and trying the same test case in What4 will not terminate in a reasonable amount of time, regardless of which Z3 version is used. A pity.

Reply:
Ahh, OK.